examples/mining_ar.py

```diff
@@ -0,0 +1,110 @@
+
+    print(f'\n\nNow lets see some results when {COLOR_CODES["yellow"]}minconf=0.7{COLOR_CODES["default"]}')
+    algo.execute(minconf=0.7)
+    print_ars(algo.get_ars())
+    print("\n['Milk'] -> ['Eggs'] with confidence 0.75 means that when milk is found in the receipt, there will be "
+          f'eggs with 75% chance. So, customers are {COLOR_CODES["bold_yellow"]}likely{COLOR_CODES["default"]} '
+          'buy eggs with the milk, but {COLOR_CODES["bold_red"]}not always.{COLOR_CODES["default"]}')
+
+    print('\nLet us turn to the next issue. You can observe that there are too many association rules found in this small dataset. '
+          'That happens because did not set up the support value. Due to this fact, the system'
+          ' we discovers all association rules, even those who present only once in the dataset. Now lets see the results with '
+          f'{COLOR_CODES["yellow"]}minsup=0.5{COLOR_CODES["default"]} and {COLOR_CODES["yellow"]}minconf=0.5{COLOR_CODES["default"]}.\n')
+    algo.execute(minsup=0.5, minconf=0.5)
+    print_ars(algo.get_ars())
+    print('\nNow you can see the number of association rules significantly decreased. '
+          'That happened since minsup=0.5, which basically says that association rule needs '
+          'to present in 50% of all transactions in order to be included in the result. '
+          'Unfortunately, you can\'t obtain support for each individual association rule with Desbordante.\n'
+          '\nA typical approach to controlling the algorithm is to employ "usefulness", which is defined as confidence * support. '
+          'In last example we set up min "usefulness" = 0.5 * 0.5 = 0.25. \n\nNow lets try with '
+          f'{COLOR_CODES["green"]}minsup=0.7{COLOR_CODES["default"]} {COLOR_CODES["green"]}minconf=0.7{COLOR_CODES["default"]}, '
+          f'so {COLOR_CODES["bold_yellow"]}"usefulness"=0.49{COLOR_CODES["default"]}\n')
+    algo.execute(minsup=0.7, minconf=0.7)
+    print_ars(algo.get_ars())
+    print('\nSo, now the total number of returned association rules is only four. '
+          'We efficiently reduced the amount of "noisy" information in our output. '
+          'However, this is just a single example of how you can evaluate the usefulness of data. '
+          'You are free to play with these parameters to see how it changes things. '
+          'Eventually, you will find out what fits your dataset and your task best.')
+
+
+def scenario_singular():
+    algo = desbordante.ar.algorithms.Default()
+    algo.load_data(table=(TABLE_SINGULAR, ',', True), input_format='tabular')
+    algo.execute()
+    table = pandas.read_csv(TABLE_SINGULAR, header=None, index_col=0)
+    #help(algo)
+
+    print('\nFor the second example lets take a dataset with the same receipts from '
+          'the same supermarket with input_format="singular". This is two-column format, where '
```

Comment:
Article

Reply:
I didn`t realize which word in this line I needed to replace with "article".

Reply:
The sentence lacks article, as in "This is a two-column format"